.checkstyle

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is ignored in the .gitignore and still here. Please remove from the PR.